### ISSUE TITLE
Would be nice to be able to add files in batches to comprehensive upload

### ISSUE THREAD

**Issue by arjunrajlab** [arjunrajlaboratory/NimbusImage#707]
Right now, when you upload, sometimes you have to carefully select particular files in the file browser, then drag them on to the dropzone:

<img width="600" alt="image" src="https://github.com/Kitware/UPennContrast/assets/13998824/70aae629-5daa-4b24-ab53-190852330009">

Sometimes people want to add one batch of files, then a different batch of files, all for the same dataset. For instance, red_001-red_010, then also green_001-green_010. One option could be to allow dropping multiple files onto the dropzone and have a "proceed" button. The other option, which I think I like more, is to add a "add more files" button to the next screen that allows you to add more files to the dataset.

**Comment by bruyeret** [comment]
After discussion : a button to add more files would be enough

**Comment by arjunrajlab** [comment]
Yes, on this screen, just add a button to bring up a file browser to allow additional files to be added. Hopefully then we don't need to manipulate any girder components, etc.

<img width="1231" alt="image" src="https://github.com/Kitware/UPennContrast/assets/13998824/f7e0952d-055e-4215-bee5-62efd3bca15d">

**Comment by arjunrajlab** [comment]
https://github.com/arjunrajlaboratory/NimbusImage/pull/857
